### DISCUSSION TITLE
button as a panel in the insight module for user to trigger API call

### DISCUSSION THREAD

**Discussion by xu4wang** [directus/directus#17511] [Archive]
I love the convenient insight module for building dashboard. I am thinking to add some more functionality for the viewer of the dashboard.  

Is it possible to implement a button panel, once clicked, it will: 
- Call the API and 
- Update/Refresh the dashboard

In this way, for the non-tech user, they can just use several customized dashboard in their daily life, no need to touch the collection module.

BR,Austin

**Comment by rijkvanzanten** [comment]
You can create that through [custom insights panels ](https://docs.directus.io/extensions/panels.html). To suggest this for inclusion in the core platform, please [open a new Feature Request](https://github.com/directus/directus/discussions/new?category=feature-requests) 🙂

**Reply by xu4wang** [reply]
> You can create that through [custom insights panels ](https://docs.directus.io/extensions/panels.html). To suggest this for inclusion in the core platform, please [open a new Feature Request](https://github.com/directus/directus/discussions/new?category=feature-requests) 🙂

THanks @rijkvanzanten Is there mechanism I can trigger a dashboard refresh from a panel or a flow?
